watertap/unit_models/tests/test_gac.py

```diff
@@ -671,63 +687,141 @@ def test_multi_solve(self, gac_frame_multi):
     def test_multi_var_scaling_solve(self, gac_frame_multi):
         mm = gac_frame_multi
         badly_scaled_var_lst = list(
-            badly_scaled_var_generator(mm, large=1e2, small=1e-2, zero=1e-8)
+            badly_scaled_var_generator(mm, large=sv_large, small=sv_small, zero=sv_zero)
         )
         assert badly_scaled_var_lst == []
 
     @pytest.mark.component
     def test_multi_solution(self, gac_frame_multi):
         mm = gac_frame_multi
 
-        # values calculated independently and near to those reported in Crittenden, 2012
-        assert pytest.approx(1.139, rel=1e-3) == value(mm.fs.unit.mass_throughput)
-        assert pytest.approx(12830000, rel=1e-3) == value(mm.fs.unit.elap_time)
-        assert pytest.approx(0.8333, rel=1e-3) == value(mm.fs.unit.bed_length)
-        assert pytest.approx(10.68, rel=1e-3) == value(mm.fs.unit.bed_area)
+        # only checking for variables new to configuration options
+        assert pytest.approx(2.473, rel=1e-3) == value(mm.fs.unit.N_Re)
```

Comment:
That's a very low value for Reynolds number... do we expect Re values to be that low in GAC?

Reply:
Just scanned an excerpt from [Crittenden et al., 1987](https://www.jstor.org/stable/41290795), that mentions a range of 0.08 to 150, so I guess the answer is yes.

Reply:
Up to 25 minute EBCT, definitely expecting laminar and low Re. Also only constructed when using that correlation, and not used in the performance equations.